doc/cloud/cloud-welcome.rst

```diff
@@ -0,0 +1,39 @@
+Welcome
```

Comment:
Is this the better name for this section?

Reply:
It was renamed to "Introduction".